TicTacToe/TicTacToe.js

```diff
@@ -0,0 +1,157 @@
+class Game {
+    moves = 0;
```

Comment:
Interesting. I wasn't aware of this syntax (apparently called ["public instance fields"](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Public_class_fields) -- JS has evolved since I last really used it).

I see that you're using it to set some default fields, but then those are copied in the `reset()` method. As an alternative (to reduce duplication), I would suggest using a constructor that calls `reset()`:
```javascript
class Game {
  constructor() {
    this.reset();
  }

  reset() {
    // ...
  }

  // ...
}
```

Or simply reduce code duplication by having constants that define initial conditions and reusing those variables rather than duplicating the values, e.g.:
```javascript
const DEFAULT_USER = 'X';
const // ...;

class Game {
  user = DEFAULT_USER;
  // ...

 reset() {
    this.user = DEFAULT_USER;
    // ...
  }

  // ...
}
```